lib/widgets/item_cocktail.dart

```diff
@@ -8,40 +10,68 @@ class CocktailItem extends StatelessWidget {
   const CocktailItem({
     Key? key,
     required this.name,
+    this.isTypeOfGlass = false,
   }) : super(key: key);
 
   final String name;
+  final bool? isTypeOfGlass;
```

Comment:
Why does this bool needs to be optional? With a default value it doesn't need to be optional, or is there a case for it when it is actually `null`?

Reply:
There is a case for it when is null